QuasarHelios2015/src/org/team1540/quasarhelios/HeadingSensor.java

```diff
@@ -32,6 +39,26 @@ public static void setup() {
 
         zeroGyro = sensor.zeroGyro;
 
+        absoluteYaw = FloatMixing.addition.of(FloatMixing.multiplication.of((FloatInput) accumulator, 360), yaw);
+
+        Igneous.globalPeriodic.send(new EventOutput() {
+            public void event() {
+                float currentyaw = yaw.get();
+                float oldyaw = oldYaw.get();
```

Comment:
`oldyaw` might make sense as an instance variable on this anonymous class instead of as a FloatStatus.